anthos-bm-gcp-terraform/variables.tf

```diff
@@ -41,7 +41,7 @@ variable "region" {
 variable "zone" {
   description = "Zone within the selected Google Cloud Region that is to be used"
   type        = string
-  default     = "us-central1-a"
+  default     = "us-central1-b"
```

Comment:
Question: What was the reason for this change (`us-central1-a` to `us-central1-b`)?

Reply:
The `nvidia-tesla-t4` are not available in `us-central1-a`, the recommendation is `us-central1-b` or `us-central1-f`.